File: Day-One/Level-Two/java/LevelTwoTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertArrayEquals;


public class LevelTwoTest{

    @Test
    public void testThatMethodReturnsNewArrayOfOnlyTheDuplicateElements(){

        int[] array = {1, 2, 3, 2, 4, 3};
        int[] expected= {2, 3};
        int[] actual = LevelTwo.getDuplicate(array);
        assertArrayEquals(actual, expected);

    }
    
//    @Test
//    public void testThatMethodReturnTheTwoIndexesOfWhereTheDuplicateNumber(){
//
//        int[] array = {1, 2, 3, 2, 4, 3};
//        int[][] expected= {2{1, 3}, 3{2, 5}};
//        int[][] actual = LevelTwo.getDuplicate(array);
//        assertArrayEquals(actual, expected);
//
//    }

    @Test
    public void testThatNewArrayContainsZeroAtTheEndOfTheAray(){

        int[] array = {5, 0 , 3, 0, 2, 0};
        int[] expected = {5, 3 , 2 , 0, 0 , 0};
        int[] actual = LevelTwo.sortZero(array);
        assertArrayEquals(actual, expected);
        
    }

//End of class
}

File: Day-One/Level-Two/Javascript/level_two.test.js
const { get_duplicate, push_zero } = require("./level_two") 

test("test that funtion returns a new array of onlyb the dupliates", ()=> {

    let array = [1, 2, 3, 2, 4, 3]
    let actual = get_duplicate(array)
    let expected = [ 2, 3 ]
    
    expect(actual).toEqual(expected)
})

test("test that array is osrted and zeros are puahed to the left", ()=> {

    let array = [5, 0 , 3, 0, 2, 0]
    let expected =  [5, 3 , 2 , 0, 0 , 0]
    let actual = push_zero(array)
    
    expect(actual).toEqual(expected)

})
